src/_utils/item/style.ts

```diff
@@ -1,19 +1,40 @@
 import css from 'styled-jsx/css'
 
-import { space } from '_utils/branding'
+import { color, space } from '_utils/branding'
 
 export default css`
   :global(.kirk-item) {
     position: relative;
     display: flex;
     padding: 0;
-    padding-top: ${space.l};
-    padding-bottom: ${space.l};
+    padding: ${space.l} ${space.xl};
```

Comment:
This is huge, it's going to impact all the Items throughout the lib (not only the `ItemAction`). Not a problem per se, just wanted to point that out.

Reply:
Yep I know that is why I listed it in the breaking changes